### PR TITLE
[doc] Update PR template

### PR DESCRIPTION
**Pull request by saimidu**
*Issue #, if available:*

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]

*Description:*
Update Pull Request template to increase readability and make it easier to use

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*
Example of new template can be seen at https://github.com/saimidu/deep-learning-containers/blob/update_pr_template/.github/PULL_REQUEST_TEMPLATE.md

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
 *Issue #, if available:*
```
Comment (jeet4320) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Lets say Github issue #

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+### Description
+
+### Tests run
+
+### DLC image/dockerfile
+
+### Additional context
+
+## Label Checklist
+- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")
```
Comment (jeet4320) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: then labels should be present for that project, some may not have access to create those.

Comment (saimidu) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: How do you propose we go about this, apart from creating the labels whenever a new project starts?

**Review by tejaschumbalkar — APPROVED** [review]
